bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -783,6 +763,19 @@ lnd/clm2/surfdata_map/surfdata_ne30np4_78pfts_CMIP6_simyr1850_c170824.nc</fsurda
 <fsurdat hgrid="ne120np4"     sim_year="1850" use_crop=".true." >
 lnd/clm2/surfdata_map/surfdata_ne120np4_78pfts_CMIP6_simyr1850_c170824.nc</fsurdat>
 
+<!-- SSP-RCP scenarios -->
```

Comment:
Actually remove this set. We don't need to have a separate 1850 fsurdat file for the SSP's from the historical one. It's good to run cprnc on the two files to verify that they aren't different though. They should be identical.

Reply:
Removed the 1850 surfdat files and associated comments.